docs/cedarling/cedarling-policy-store.md

```diff
@@ -124,125 +88,184 @@ Here is a non-normative example of the `policies` field:
-      "policy_content": {
-        "encoding": "base64",
-        "content_type" : "cedar",
-        "body": "kJW1bWl0KA0g3CAxa..."
-      }
+      "policy_content": "kJW1bWl0KA0g3CAxa..."
     },
     ...
   }
 ```
 
-## Identity Source Schema
+## Trusted Issuers Schema
 
 This record contains the information needed to validate tokens from this issuer:
 
 ```json
-  "identity_source": {
-    "some_unique_id" : {
-      "name": "name_of_the_trusted_issuer",
-      "description": "description for the trusted issuer",
-      "openid_configuration_endpoint": "https://<trusted-issuer-hostname>/.well-known/openid-configuration",
-      "access_tokens": {
-        "trusted": true,
-        "principal_identifier": "",
-        "role_mapping": "",
-      },
-      "id_tokens": {
-        "trusted": true,
-        "principal_identifier": "sub",
-        "role_mapping": "",
-      },
-      "userinfo_tokens": {
-        "trusted": true,
-        "principal_identifier": "",
-        "role_mapping": "role",
-      },
-      "tx_tokens": {
-        "trusted": true,
-        "principal_identifier": "",
-        "role_mapping": "",
-      },
-    }
-    ...
-  }
+"trusted_issuers": {
+  "some_unique_id" : {
+    "name": "name_of_the_trusted_issuer",
+    "description": "description for the trusted issuer",
+    "openid_configuration_endpoint": "https://<trusted-issuer-hostname>/.well-known/openid-configuration",
+    "access_tokens": { 
+      "trusted": true,
+      "principlal_identifier": "jti",
+      ...
+    },
+    "id_tokens": { ... },
+    "userinfo_tokens": { ... },
+    "tx_tokens": { ... },
+  },
+  ...
+}
 ```
 
 - **name** : (*String*) The name of the trusted issuer.
 - **description** : (*String*) A brief description of the trusted issuer, providing context for administrators.
 - **openid_configuration_endpoint** : (*String*) The HTTPS URL for the OpenID Connect configuration endpoint (usually found at `/.well-known/openid-configuration`).
 - **identity_source** : (*Object*, *optional*) Metadata related to the tokens issued by this issuer.
+- **`access_tokens`, `id_tokens`, `userinfo_tokens`, `tx_tokens`** : (*Object*, *optional*) Metadata related to the tokens issued by this issuer. See schema [below](#token-entity-metadata-schema). **NOTE**: The `access_tokens` field contains the additional fields: `trusted` and `principal_identifier` in addition to the fields from it's `TokenEntityMetadata`.
 
-### Token Metadata Schema
+
+### Token Entity Metadata Schema
+
+The Token Entity Metadata Schema defines how tokens are mapped, parsed, and transformed within Cedarling. It allows you to specify how to extract user IDs, roles, and other claims from a token using customizable parsers.
 
 ```json
 {
-  "trusted": true|false
-  "principal_identifier": "some_user123",
-  "role_mapping": "role",
+  "token_type": {
+    "user_id": "<field name in token (e.g., 'email', 'sub', 'uid', etc.) or '' if not used>",
```

Comment:
Let's use the `claim name` instead of the field name to show the exactness with the OIDC standard.

ref: https://openid.net/specs/openid-connect-basic-1_0.html#IDToken

Reply:
i was just following the wiki: [Token Entity Metadata Schema](https://github.com/JanssenProject/jans/wiki/Cedarling-Nativity-Plan#token-entity-metadata-schema)... maybe a discussion should be made first for this one? I thought we should be just following the wiki because discussions have been made for that already.